cognite/client/data_classes/filters.py

```diff
@@ -185,8 +185,13 @@ def load(cls, filter_: dict[str, Any]) -> Filter:
                 property=filter_body["property"],
                 value=_load_filter_value(filter_body["value"]),
             )
+        elif filter_name == InvalidFilter._filter_name:
```

Comment:
Do not think I get this case? When will this happen? I do not see any test hitting it either.

Reply:
When a filter references resources which no longer exist there are cases when we don't have necessary information to map these filters to the necessary external representation, so we just need to use a "invalid" placeholder to indicate to the user that they need to fix their filter.